src/Common/ThreadStatus.cpp

```diff
@@ -88,7 +88,7 @@ struct ThreadStack
         auto size = std::max<size_t>(UNWIND_MINSIGSTKSZ, MINSIGSTKSZ);
 
         if constexpr (guardPagesEnabled())
-            size += 1;
+            size += getPageSize();
```

Comment:
Note, that this should not affect anything due to `aligned_alloc`, that should implicitly add one more page